src/Sylius/Bundle/ApiBundle/CommandHandler/Cart/BlameCartHandler.php

```diff
@@ -51,6 +51,7 @@ public function __invoke(BlameCart $blameCart): void
         }
 
         $cart->setCustomer($user->getCustomer());
+        $cart->setCreatedByGuest(false);
```

Comment:
Maybe we should set this flag together with the customer in some `assignLoggedInCustomer` method inside the `Order`? 🤔 That would make the object responsible for its behaviour. But it's definitely an improvement we can introduce later 🖖

Reply:
I will open a PR with proposal based on this one

Reply:
https://github.com/Sylius/Sylius/pull/13968